autorest/codegen/models/imports.py

```diff
@@ -52,14 +52,22 @@ def __hash__(self):
                 retval += hash(getattr(self, attr))
         return retval
 
+class TypeDefinition:
+    def __init__(self, sync_definition: str, async_definition: str,
+                version_if_else_imports: Optional[Tuple[Tuple[int, int], ImportModel, ImportModel]] = None):
+        # version_if_else_imports: Tuple[version_info, imports if >= version_info, imports else]
+        self.sync_definition = sync_definition
+        self.async_definition = async_definition
+        self.version_if_else_imports = version_if_else_imports
```

Comment:
I like the idea of making this a model! I think for `version_if_else_imports`, it'll be easier to to understand if it's some kind of mapping between version requirements, and which import to do, instead of nested tuples

Reply:
yes, now updated to use a map to present version->importModel